Library/HUMSlider.m

```diff
@@ -19,14 +21,33 @@
 static CGFloat const HUMImagePadding = 8;
 
 // Sizes
-static CGFloat const HUMTickHeight = 6;
+static CGFloat const HUMTickHeight = 8;
 static CGFloat const HUMTickWidth = 1;
 
+@implementation Tick
+// Constructor for a tick
+- (id)initWithPosition:(double)position {
+    NSAssert(position >= 0 && position <= 1, @"Position must be between 0 and 1");
+    self = [super init];
+    if (self) {
+        self.position = position;
+    }
+    return self;
+}
+@end
+
 @interface HUMSlider ()
+
 @property (nonatomic) NSArray *tickViews;
 @property (nonatomic) NSArray *allTickBottomConstraints;
-@property (nonatomic) NSArray *leftTickRightConstraints;
-@property (nonatomic) NSArray *rightTickLeftConstraints;
+
+//Constraint storage for evenly spaced tick constraints.
+@property (nonatomic) NSArray *leftTickRightConstraints; //Confusing name.. what is a left tick? Left of middle?
```

Comment:
This an array of the constraints constraining the left side of the ticks. The other is an array of the constraints constraining the right side of the ticks.